PayloadPkg/OsLoader/BootOption.c

```diff
@@ -52,9 +52,30 @@ PrintBootOptions (
                  BootOption->Image[0].LbaImage.LbaAddr \
                  ));
     }
-  }
+    //Print Pre-OS image filename
+    if ((BootOption->BootFlags & BOOT_FLAGS_PREOS) == BOOT_FLAGS_PREOS){
+        if (BootOption->Image[LoadImageTypePreOs].LbaImage.Valid == 1) {
+          ShellPrint (L" [%x|0x%x]", BootOption->Image[LoadImageTypePreOs].LbaImage.SwPart, \

```

Comment:
Should you use DEBUG instead of ShellPrint in this file?

Reply:
yes i missed out that part. Submitted patch.